assets/js/src/segments/list.jsx

```diff
@@ -104,7 +108,7 @@ const itemActions = [
       );
     },
     display: function display(segment) {
-      return (segment.type !== 'wp_users');
+      return isWPUsersSegment(segment);
```

Comment:
I think this should be `!isWPUsersSegment(...)`. Or maybe even `!isSpecialSegment(...)` but that we can fix also by another pull request (it it's a bug).

Reply:
Fixed.